aws/data_source_aws_vpc_ids.go

```diff
@@ -0,0 +1,63 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsVpcIDs() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsVpcIDsRead,
+		Schema: map[string]*schema.Schema{
+			"filter": dataSourceFiltersSchema(),
```

Comment:
A few items with this attribute:
* It does seem to be read into `req.Filters` so it appears to not currently be doing anything
* Its missing in the data source documentation

I would recommend testing this attribute separately from the `_basic` test (preferably not being passed any arguments) and the `_tags` test (preferably only testing tags filtering).

Reply:
Its my mistake, I added the filter in last minute when I was working on test cases, then I forgot to follow on.